ui/README.md

```diff
@@ -51,6 +51,11 @@ that are static and are re-loaded on page refresh.
 - `ember serve`
 - The app will be available at [http://localhost:4200](http://localhost:4200).
 
+Troubleshooting:
+
+- If you run into issues with `ember serve`, try deleting the directory `ui/node_modules/` and rerunning an `ember serve`.
+- You will need a Waypoint CLI token to log in, e.g. the output of `waypoint user token`. To achieve this, run Docker and the Waypoint CLI.
```

Comment:
In mirage-mode, this can actually be any string at all

Reply:
Maybe this could read:

> You can use literally any non-empty string as the login token, i.e. `my-cool-token`.

Reply:
Thinking about this some more, we should probably move this instruction up into the list above, i.e.:

```md
This returns data in-browser with Mirage.js (a mocking framework)
active. This means that the network requests will be intercepted
and return [mocked objects](https://github.com/hashicorp/waypoint/tree/master/ui/mirage/services)
that are static and are re-loaded on page refresh.

- `ember serve`
- The app will be available at [http://localhost:4200](http://localhost:4200).
- When prompted for a token, you can use any non-empty string (i.e. `my-cool-token`)
```

Reply:
Just tried it on my machine - thanks both! Updated the README with ^^, LMK what you think